packages/administration/templates/form/theme.html.twig

```diff
@@ -71,3 +71,34 @@
 
     {{ parent('number_widget') }}
 {% endblock number_widget %}
+
+{%- block form_widget_compound -%}
```

Comment:
We don't need to change the behaviour of all compound widgets. We can set:

```
if ($form->getParent() === null) {
    $view->vars['block_prefixes'][] = 'auto_group';
}
```

and then use:

```
{% block auto_group_widget %}
    ...
{% endblock %}
```

What do you think?